nbterm/cell.py

```diff
@@ -178,6 +178,9 @@ def set_input_readonly(self):
         elif self.json["cell_type"] == "code":
             code = Syntax(self.input_buffer.text, self.notebook.language)
             text = rich_print(code)[:-1]  # remove trailing "\n"
+        elif self.json["cell_type"] == "raw":
+            text = self.input_buffer.text or "Type *Raw*"
+            text = text[:-1]  # remove trailing "\n"
```

Comment:
Do we need to remove the trailing "\n"?

Reply:
I think so, because otherwise we would end up with two newlines when saving the raw cell back to the file.

Reply:
Hold on, I think this needs some more review whether the "-1" is really needed, sorry about the confusion. Will need to leave now for home, will continue tomorrow.

Reply:
Actually, I fixed it, so indeed, removing the last character was a mistake. I hope everything is fine now.